extensions/sql-database-projects/src/test/dialogs/testUtils.ts

```diff
@@ -0,0 +1,98 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as TypeMoq from 'typemoq';
+import * as vscodeMssql from 'vscode-mssql';
+import { RequestType } from 'vscode-languageclient';
+
+export interface TestUtils {
+	vscodeMssqlIExtension: TypeMoq.IMock<vscodeMssql.IExtension>;
+}
+
+export class MockVscodeMssqlIExtension implements vscodeMssql.IExtension {
```

Comment:
sql bindings also has something similar to this. It would be nice for this to be in https://github.com/microsoft/azdata-test so that it doesn't have to be copied and maintained separately in these two extensions. Not something you have to do now, but it would be a nice cleanup thing to do eventually